dotenv-linter/tests/common/test_dir.rs

```diff
@@ -118,7 +131,7 @@ impl TestDir {
         T: Into<String>,
     {
         let expected_output = expected_output.into();
-        let mut cmd = Self::init_cmd();
+        let mut cmd = Self::init_cmd(self.envs.clone());
```

Comment:
Btw, looks like you can avoid extra cloning in two ways:
1. Inplace env command, to make it like this:
```
let mut cmd = Self::init_cmd();
cmd.envs(&self.envs);
````
Looks like it should work.

But second option could be even better, declare function `init_cmd` as method. So it will be: `self.init_cmd();` and will pass envs as reference to cmd

Reply:
Thanks! Updated here: https://github.com/dotenv-linter/dotenv-linter/pull/625/commits/c64b3667d04861528bf4a93e506ffca117fb4e07

Reply:
Appreciate all the suggestions by the way, really helpful for me to get myself more familiar with the language! :)